bin/cron/deliver_poste_messages

```diff
@@ -246,7 +247,16 @@ class Deliverer
   end
 
   def parse_addresses(string)
-    string.split(', ').map do |user_string|
+    # Split on the commas separating separate email addresses, but don't
+    # split when they're inside a pair of double-quotes.
+    # e.g. This will split
+    #        "\"friendly, name\" <name1@code.org>, \"other name\" <name2@code.org>"
+    #      into
+    #        ["\"friendly, name\" <name1@code.org>", " \"other name\" <name2@code.org>"]
+    #
+    # Adapted from https://stackoverflow.com/a/18893443.
+    # Note that this is still problematic when a single double-quote appears.
+    string.split(/,(?=(?:[^\"]*\"[^\"]*\")*[^\"]*$)/).map do |user_string|
```

Comment:
Another option to consider, we take out any strings in double quotes, sanitize them (removing comma, single quote, etc.) and then continue to use the previous logic.

Reply:
That way we only have to test the sanitizier function.